apps/signup/signup.html

```diff
@@ -32,29 +50,41 @@
       if (attr == "2"){
         userType = "Editor"
       }
-      var doc = {email: email, userType: userType, userFilter:filters}
-      fetch(userSignupUrl, {
-          method: 'POST',
-          mode: 'cors', // no-cors, cors, *same-origin
-          cache: 'no-cache', // *default, no-cache, reload, force-cache, only-if-cached
-          credentials: 'same-origin', // include, *same-origin, omit
-          headers: {
-              'Content-Type': 'application/json',
-          },
-          body: JSON.stringify(doc)
-      }).then(x=>{
-        if (x.status>=400){
-          throw "failed to sign up user"
-        }
-        x.json()
-      }).then(x=>{
-        window.alert("success")
-        console.log(x)
-      }).catch(e=>{
-        // window.alert("error!")
-        console.error(e)
-    })
+      // if (getUserType() === "Admin") {
+      if (permissions.user.post == true) {
```

Comment:
I'm getting a permissions not defined error here.

Reply:
Ah sorry, I might've missed it. Yeah done.